Lesson32/feyza/my-app/src/shared-components/typography/Display/index.jsx

```diff
@@ -0,0 +1,5 @@
+import "./styles.css";
+
+export const Display = ({ titleDisplay }) => {
+  return <h1 className="display-title">{titleDisplay} Display</h1>;
```

Comment:
Should be `return <h1 className="display-title">{titleDisplay}</h1>;` (or {title} after you update the prop name).
If you leave any hardcoded text (e.g. "Display") it will show up every time the component is used while the goal is to create a completely reusable component that can be used for any text.